src/web/app/pages-instructor/instructor-course-edit-page/instructor-course-edit-page.component.ts

```diff
@@ -228,14 +230,13 @@ export class InstructorCourseEditPageComponent implements OnInit {
       coursetimezone: newTimeZone,
     };
 
-    this.httpRequestService.put('/instructors/course/details/save', paramsMap)
-        .subscribe((resp: MessageOutput) => {
-          this.statusMessageService.showSuccessMessage(resp.message);
-          this.updateCourseDetails(newName, newTimeZone);
-          this.toggleIsEditingCourse();
-        }, (resp: ErrorMessageOutput) => {
-          this.statusMessageService.showErrorMessage(resp.error.message);
-        });
+    this.courseService.updateCourse(paramsMap).subscribe((resp: MessageOutput) => {
```

Comment:
You need to create `CreateCourseRequest`. Take a look at `feedbackQuestionService` for example.